policy/modules/system/authlogin.te

```diff
@@ -391,7 +398,6 @@ files_manage_etc_files(updpwd_t)
 term_dontaudit_use_console(updpwd_t)
 term_dontaudit_use_unallocated_ttys(updpwd_t)
 
-auth_etc_filetrans_shadow(updpwd_t)
```

Comment:
This makes me nervous. How sure are you this is correct, notwithstanding the desire to get opasswd to the right context? Otherwise, this makes sense.